src/modm/platform/core/hosted/delay.hpp.in

```diff
@@ -74,19 +77,21 @@ delay_ms(int ms)
 inline void
```

Comment:
Just above this, there is commented-out Windows code; might make sense to delete at least.